src/content/glossary/index.md

```diff
@@ -621,10 +753,18 @@ An off-chain scaling solution that uses [fraud proofs](#fraud-proof), like [Opti
   Plasma
 </DocLink>
 
+### presale {#presale}
+
+Sale of cryptocurrency before the actual launch of the network.
+
 ### private key (secret key) {#private-key}
 
 A secret number that allows Ethereum users to prove ownership of an account or contracts, by producing a digital signature (see [public key](#public-key), [address](#address), [ECDSA](#ecdsa)).
 
+### private chain {#private-chain}
+
+A fully private blockchain is one with write permissions limited to one organization.
```

Comment:
Is this accurate? I find it useful to think of private chains as what they are not (open, decentralized) rather than what they are (closed, centralized). How closed they are seems arbitrary to me.

Reply:
idk, I think this definition makes sense. One "organization" can be one person or one hundred people with multiple affiliations - the point is that not everyone can append data to that particular blockchain.

Suggest changing to:

"A permissioned blockchain with restrictions on who can add data." ?

Reply:
It's not just writing data necessarily either. If I spin up a chain on my own network with restricted access, the public cannot read it either. I could potentially share it with whoever I want though, so it's also not strictly limited to any one entity or organization. 

Could simplify to:
```suggestion
A fully private blockchain is one with permissioned access, not publicly available for use. 
```